src/www/views/contact_view/index.ts

```diff
@@ -45,15 +44,20 @@ export class ContactView extends LitElement {
   static styles = [
     css`
       :host {
+        background: #fff;
         color: var(--outline-text-color);
-        display: block;
         font-family: var(--outline-font-family);
-        margin: 0 auto;
-        max-width: var(--contact-view-max-width);
         padding: var(--contact-view-gutter, var(--outline-gutter));
         width: 100%;
       }
 
+      :host > * {
```

Comment:
Done, though I'm not a big fan of violating separation of concerns this way. HTML elements should only serve a structural purpose, not a style one.

What side effects you are worried about that wouldn't also apply to a container element? The universal selector isn't all that special; I've never run into issues with proper style encapsulation.

Reply:
RE: that third point. That shouldn't be the responsibility of the individual views at all imo. We should do that at a layer above the individual views at the page layout level in app-root. This isn't actually solving that issue (though how I had it before also didn't solve that of course, we need a larger change to move those layout concerns to app-root). So something like:

```
<main>
  <iron-pages>
    <individual view1 />
    <individual view2 />
  </iron-pages>
</main>
```

Instead of what we have now:

```
<iron-pages>
  <individual view1>
    <main />
  </individual view1>
  <individual view2>
    <main />
  </individual view2>
<iron-pages>
```

Reply:
Excellent! 😄